src/tests/Microsoft.PowerFx.Interpreter.Tests/RefreshTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.Entities;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Tests
+{
+    public class RefreshTests
+    {
+        [Fact]
+        public void RefreshTest_InMemoryTable()
+        {
+            PowerFxConfig config = new PowerFxConfig(Features.PowerFxV1);                        
+            RecalcEngine engine = new RecalcEngine(config);
+
+            config.EnableSetFunction();
+            engine.UpdateVariable("t", TableValue.NewTable(RecordType.Empty()));            
+            FormulaValue result = engine.Eval("Set(t, Table()); Refresh(t)", null, new ParserOptions { AllowsSideEffects = true });
```

Comment:
>Set(t, Table()); [](http://example.com/codeflow?start=47&length=16)

Is this necessary if set set it via hosting API on line 21?

Reply:
Not really, I'll remove it.